pom.xml

```diff
@@ -641,7 +641,13 @@
             <dependency>
                 <groupId>org.graylog2.repackaged</groupId>
                 <artifactId>grok</artifactId>
-                <version>0.1.3-graylog</version>
+                <version>0.1.6-graylog</version>
+            </dependency>
+            <dependency>
+                <groupId>com.github.tony19</groupId>
+                <artifactId>named-regexp</artifactId>
```

Comment:
What is this dependency used for? It's also already a transitive dependency of `org.graylog2.repackaged:grok:0.1.6-graylog`.